debug_scripts/create-new-eeprom-with-self-signed-certs.sh

```diff
@@ -50,6 +51,7 @@ execute () {
         -g $lwm2mserveruri\
         -p DEV0\
```

Comment:
Saw this in passing. I assume that is is meant to be fixed, and not a varaible?

Reply:
Ya its not a variable, it will be used in case there is no serial number provided. That is also an indication that we are in dev mode so lets generate a dummy serial number.